From cebca0ff9360247812d2270e3814cb64803f37f3 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Thu, 27 Jul 2017 18:21:56 +0800
Subject: [PATCH 2252/5242] MLK-16062-1: Fix PXL mipi csi0/1 clock gate
 register address

commit  8e076cc38792f795248417f45a07403ec630698d from
https://source.codeaurora.org/external/imx/linux-imx.git

mipi csi0/1 clock gate register address swapped.
It will cause mipi csi0/1 failed to work.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/soc/imx8/imx8qm/lpcg.h  |    4 ++--
 include/soc/imx8/imx8qxp/lpcg.h |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/soc/imx8/imx8qm/lpcg.h b/include/soc/imx8/imx8qm/lpcg.h
index c26c3b4..512953e 100644
--- a/include/soc/imx8/imx8qm/lpcg.h
+++ b/include/soc/imx8/imx8qm/lpcg.h
@@ -102,8 +102,8 @@
 #define IMG_PXL_LINK_DC1_LPCG	0x585C0000
 #define IMG_PXL_LINK_DC0_LPCG	0x585B0000
 #define IMG_PXL_LINK_HDMI_LPCG	0x585A0000
-#define IMG_PXL_LINK_CSI0_LPCG	0x58590000
-#define IMG_PXL_LINK_CSI1_LPCG	0x58580000
+#define IMG_PXL_LINK_CSI1_LPCG	0x58590000
+#define IMG_PXL_LINK_CSI0_LPCG	0x58580000
 #define IMG_PDMA_7_LPCG			0x58570000
 #define IMG_PDMA_6_LPCG			0x58560000
 #define IMG_PDMA_5_LPCG			0x58550000
diff --git a/include/soc/imx8/imx8qxp/lpcg.h b/include/soc/imx8/imx8qxp/lpcg.h
index 061454f..cb06b03 100644
--- a/include/soc/imx8/imx8qxp/lpcg.h
+++ b/include/soc/imx8/imx8qxp/lpcg.h
@@ -99,8 +99,8 @@
 #define IMG_PXL_LINK_DC1_LPCG	0x585C0000
 #define IMG_PXL_LINK_DC0_LPCG	0x585B0000
 #define IMG_PXL_LINK_HDMI_LPCG	0x585A0000
-#define IMG_PXL_LINK_CSI0_LPCG	0x58590000
-#define IMG_PXL_LINK_CSI1_LPCG	0x58580000
+#define IMG_PXL_LINK_CSI1_LPCG	0x58590000
+#define IMG_PXL_LINK_CSI0_LPCG	0x58580000
 #define IMG_PDMA_7_LPCG			0x58570000
 #define IMG_PDMA_6_LPCG			0x58560000
 #define IMG_PDMA_5_LPCG			0x58550000
-- 
1.7.9.5

